expanding acronyms in content (#2671)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/initial-teacher-training.md

School placements vary in length and structure, but the minimum length is usually around 6 weeks. You should check with your training provider what their course and placement patterns are.

> Read about [what teacher training is like doing a QTS with PGCE at the University of Cumbria](/blog/what-to-expect-on-your-teacher-training).

Reviewer: ```suggestion
Read about [what teacher training is like doing a QTS with PGCE with one training provider](/blog/what-to-expect-on-your-teacher-training).
```

should we make this more generic to not promote Cumbria?
